packages/bsky/src/api/com/atproto/repo/getRecord.ts

```diff
@@ -18,12 +18,22 @@ export default function (server: Server, ctx: AppContext) {
       throw new InvalidRequestError(`Could not locate record: ${uri}`)
     }
 
+    const labelers = ctx.reqLabelers(req)
+    const labelMap = await ctx.hydrator.label.getLabelsForSubjects(
+      [uri],
+      labelers,
+    )
+    const labels = Array.from(labelMap.get(uri)?.labels?.values() || [])
```

Comment:
Hydrating labels here in `repo.getRecord` is a little bit funky since it's quietly off-contract (and I don't think we'll want to add it to the contract).  I believe we had plans for the appview to implement `com.atproto.label.queryLabels` to provide the labels on arbitrary subjects, which I think would speak to ozone's use case.  Perhaps we could implement that now, I don't think it's going to be super big lift.

@bnewbold what do you think?